services/tests.py

```diff
@@ -450,3 +483,62 @@ def test_api_get_user_token_two_services(self):
         self.assertEqual(tokens.count(), 2)
         self.assertEqual(tokens[0].token, "fakertoken")
         self.assertEqual(tokens[1].token, "otherfakertoken")
+
+    @responses.activate
+    def test_task_service_metric_sync(self):
+        # Setup
```

Comment:
Sorry, just show that there's 0 beforehand? It'll make it clearer what this task is doing.